client/webserver/site/src/js/charts.js

```diff
@@ -83,8 +83,8 @@ export class DepthChart {
     }
 
     bind(this.canvas, 'wheel', e => { this.wheel(e) })
-    this.resize = h => { this.resize_(h) }
-    bind(window, 'resize', () => { this.resize(parent.clientHeight) })
```

Comment:
I had a feeling it had something to do with these and `parent` changing, but these lines are making my head spin.